Makefile

```diff
@@ -196,13 +199,24 @@ install: $(DEBUG_APK) ## Install the debug APK on a connected device
 run: install ## Run the debug APK on a connected device
 	adb shell am start -n com.tailscale.ipn/com.tailscale.ipn.MainActivity
 
-.PHONY: dockershell
-dockershell: ## Run a shell in the Docker build container
-	docker build -t tailscale-android .
-	docker run -v $(CURDIR):/build/tailscale-android -it --rm tailscale-android
+.PHONY: dockerEnv
+dockerEnv: ## Builds the docker image for the android build environment
```

Comment:
we have kebab case and snake case and uncased concatenations in this makefile already, the traditional choice is kebab case or uncased concatenation - but I'd really love it if we don't introduce a fourth style. CamelCase is extremely rare in make because it breaks down on case insensitive filesystems.

Reply:
Perhaps:

docker-image:

docker-build:

docker-remove-image:

Reply:
Fixed.